libraries/AP_Mount/AP_Mount_SToRM32.cpp

```diff
@@ -26,64 +26,59 @@ void AP_Mount_SToRM32::update()
         // move mount to a "retracted" position.  To-Do: remove support and replace with a relaxed mode?
         case MAV_MOUNT_MODE_RETRACT: {
             const Vector3f &target = _params.retract_angles.get();
-            _angle_rad.roll = radians(target.x);
-            _angle_rad.pitch = radians(target.y);
-            _angle_rad.yaw = radians(target.z);
-            _angle_rad.yaw_is_ef = false;
+            mnt_target.angle_rad.set(ToRad(target.x), ToRad(target.y), ToRad(target.z), false);
```

Comment:
Use a vector instead of x,y,z separately.

You can multiply the target vector by `DEG_TO_RAD`